.github/workflows/docat.yml

```diff
@@ -3,7 +3,7 @@ name: docat ci
 on: [push]
```

Comment:
```suggestion
on: [push, pull_request]
```

I think we require a `pull_request` trigger, too.

Reply:
sure i checked the docs and idk lets add it